source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -354,24 +354,12 @@ namespace PlayFab
 
         localbatch.clear(); // batch vector will be reused
         localbatch.reserve(this->settings->maximalNumberOfItemsInBatch);
-        if (this->settings->emitType == PlayFabEventPipelineType::PlayFabPlayStream)
-        {
-            // call Events API to send the batch
-            eventsApi->WriteEvents(
-                batchReq,
-                std::bind(&PlayFabEventPipeline::WriteEventsApiCallback, this, std::placeholders::_1, std::placeholders::_2),
-                std::bind(&PlayFabEventPipeline::WriteEventsApiErrorCallback, this, std::placeholders::_1, std::placeholders::_2),
-                customData);
-        }
-        else
-        {
-            // call Events API to send the batch, bypassing Playstream
-            eventsApi->WriteTelemetryEvents(
```

Comment:
Once upon a time, this separation was deliberate, and quite important. TelemetryEvents don't do all the same PlayStream post-processing, and are therefore SIGNIFICANTLY faster in many ways.

Reply:
I'm not sure why this is being removed. Please explain.

Reply:
I incorrectly assumed that WriteEvents & WriteTelemetryEvents were consolidated in the cerulean versions as the latter is not included in the cerulean APIs. I have filed a bug to track this: https://dev.azure.com/microsoft/Xbox/_sprints/taskboard/GDX.SDKs.Core/Xbox/22-4W20%20(Ends%205-13)/2W18%20(Ends%204-29)?workitem=39141518

once this is solved, I'll get back to adding this once again